packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -567,7 +632,7 @@ class Border extends BoxBorder {
     assert(() {
       if (!_strokeAlignIsUniform || top.strokeAlign != BorderSide.strokeAlignInside) {
         throw FlutterError.fromParts(<DiagnosticsNode>[
-          ErrorSummary('A Border can only draw strokeAlign different than BorderSide.strokeAlignInside on uniform borders.'),
+          ErrorSummary('A Border can only draw strokeAlign different than BorderSide.strokeAlignInside on borders with uniform colors.'),
```

Comment:
`uniform colors/styles` instead? Uniform colors is way more common and `style == BorderStyle.none` is kind of an invisible color.

Reply:
Thanks!